pkg/controller/clusterconnection/clusterconnection_controller.go

```diff
@@ -422,6 +446,7 @@ func (r *ReconcileConnection) Reconcile(ctx context.Context, request reconcile.R
 		TrustedCertBundle:           trustedBundle,
 		ManagementClusterConnection: managementClusterConnection,
 		GuardianClientKeyPair:       guardianKeyPair,
+		CnxVersion:                  cnxVersion,
```

Comment:
```suggestion
		Version:                  cnxVersion,
```

`CNX` isn't really a thing. Best to avoid using it where we can. I know it's still in the ClusterInformation API (ideally we'd get rid of / rename that field at some point).